kfs-core/src/main/java/edu/arizona/kfs/fp/businessobject/ErrorCertification.java

```diff
@@ -51,7 +51,7 @@ public String getErrorCorrectionReason() {
 	public void setErrorCorrectionReason(String errorCorrectionReason) {
 		this.errorCorrectionReason = errorCorrectionReason;
 	}
-	
+
```

Comment:
Yes, I tried as hard as possible to avoid it, but when having to deal with the multiple merge conflicts from having pulled in effectively ~20 other commits from the ua-development branch, a couple formatting changes did get through. I spent an entire day trying to iron it all out, and these three squashed commits were the best I could manage. While on that subject, I'd like to discuss standardizing how we format new files that are pulled in, so that we can avoid this entirely in the future.